vllm/engine/metrics.py

```diff
@@ -600,10 +638,39 @@ def _log_prometheus(self, stats: Stats) -> None:
                             stats.time_decode_requests)
         self._log_histogram(self.metrics.histogram_time_in_queue_request,
                             stats.time_in_queue_requests)
+        self._log_histogram(
+            self.metrics.histogram_time_per_prefill_token_request,
+            stats.time_per_prefill_token_requests)
         self._log_histogram(self.metrics.histogram_model_forward_time_request,
                             stats.model_forward_time_requests)
         self._log_histogram(self.metrics.histogram_model_execute_time_request,
                             stats.model_execute_time_requests)
+        # Model load time
```

Comment:
The `_get_stats()` function in `llm_engine` should format metrics properly so that this function is as clean as possible. Notice that every other metric is setup such that we can just call `_log_xxx` in this function.